### DISCUSSION TITLE
Downgrade to Minecraft 1.20.1

### DISCUSSION THREAD

**Discussion by playgamesgo** [Minestom/Minestom#2260] [Help] [answered]
How can I downgrade Minestom to 1.20.1 version, and will the documentation work under the old version?

**Comment by GoldenStack** [comment, marked as answer]
Minestom only supports the latest version. If you really want to use an old version, you could use an old commit or downgrade the protocol yourself, but keep in mind you will not receive support for this.

**Reply by playgamesgo** [reply]
Ok, where can I find last commit for Minestom 1.20.1 and for wiki?

**Reply by MelonHell** [reply]
https://github.com/Minestom/Minestom/commit/43baf51d8d45dd7149316bdccc332829c95ef44e

**Reply by playgamesgo** [reply]
Thanks
